modepy/quadrature/__init__.py

```diff
@@ -87,6 +88,18 @@ def __call__(self, f):
         return np.dot(self.weights, f(self.nodes))
 
 
+class PointQuadrature(Quadrature):
```

Comment:
I'm not sure I'm loving `PointQuadrature` as a name. (Because all quadratures have to do with points somehow...) `ZeroDQuadrature`? Also awkward, I know.

Reply:
Renamed in b0563c1.

Yeah, not sure I have anything better. `Quadrature0D`? That looks weird too..

Reply:
`ZeroDimensionalQuadrature`?

Reply:
> `ZeroDimensionalQuadrature`?

That works for me. (And I like it better than the prior names.)

Reply:
Done! 354c190